microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -12,34 +12,37 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, I
 
 # Understand WebView2 SDK versions  
 
-WebView2 depends on Microsoft Edge to function.  Each WebView2 SDK requires that a minimum browser version is installed.  The minimum version is reflected in the package version of the SDK.  For example, if you use the `SDK package version 0.9.488`, then you must install Microsoft Edge with a build number of 488 or later.  The browser version is also specified in the WebView2 [Release Notes][Releasenotes].  For more information about the latest release of the Microsoft Edge browser, navigate to [Browser Channels][DeployedgeChannels].  
+To develop a WebView2 application, a minimum version of [WebView2 Runtime]() or a [non-stable Microsoft Edge channel]() must be installed.  The minimum version is reflected in the package version of the SDK.  For example, if you use the `SDK package version 0.9.488`, then you must install [WebView2 Runtime]() or a [non-stable Microsoft Edge channel]() with a build number of 488 or later.  The browser version is also specified in the WebView2 [Release Notes][Releasenotes].  For more information about the latest release of the Microsoft Edge browser, navigate to [Browser Channels][DeployedgeChannels].    
```

Comment:
> [WebView2 Runtime]()
Can link to thttps://docs.microsoft.com/microsoft-edge/webview2/concepts/distribution#understanding-the-webview2-runtime

Reply:
Or should I link to the page where you can install WebView2 Runtime?